starport/services/network/blockchain.go

```diff
@@ -0,0 +1,252 @@
+package network
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"io"
+	"net/http"
+	"os"
+
+	launchtypes "github.com/tendermint/spn/x/launch/types"
+	profiletypes "github.com/tendermint/spn/x/profile/types"
+	sperrors "github.com/tendermint/starport/starport/errors"
+	"github.com/tendermint/starport/starport/pkg/chaincmd"
+	"github.com/tendermint/starport/starport/pkg/cosmosver"
+	"github.com/tendermint/starport/starport/pkg/events"
+	"github.com/tendermint/starport/starport/pkg/gitpod"
+	"github.com/tendermint/starport/starport/pkg/xos"
+	"github.com/tendermint/starport/starport/services/chain"
+)
+
+type Blockchain struct {
```

Comment:
Need comment for what this structure is used for

Reply:
49b25037